backend/users/tests.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.test import TestCase, Client
+from django.urls import reverse
+from rest_framework import status
+
+client = Client()
+
+
+class RegisterUserTest(TestCase):
+    """Test module for register a new user"""
+
+    def setUp(self) -> None:
+        self.valid_payload = {
+            "username": "test_user",
+            "password": "test_password",
+            "password2": "test_password",
+            "email": "test@mail.com",
+            "first_name": "test",
+            "last_name": "user",
+        }
+        self.invalid_payload = {
+            "username": "test_user2",
+            "password": "test_password2",
+            "password2": "test_password3",
+            "email": "testmail.com",
+            "first_name": "test",
+            "last_name": "",
+        }
```

Comment:
Move this data to `setTestUpData` class method, it is only called once. `setUp` is called for every test.